src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -99,6 +99,9 @@ public override ImmutableArray<Location> Locations
             }
         }
 
+        public override Location TryGetFirstLocation()
+            => _mergedDeclaration.Declarations[0].NameLocation;
```

Comment:
>Declarations[0] [](http://example.com/codeflow?start=34&length=15)

No context here, just wanted to make sure this will always exist (as well as the other unchecked array access in this commit).

Reply:
Yeah. Merged decls always have at least one subdecl